scripts/litePageSizeValidator/index.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-console */
+import { exec } from 'node:child_process';
+
+const litePageSizeValidator = async () => {
+  const urlsToCheck = [
+    '/hindi',
+    '/mundo/articles/cddylv9g8z0o',
+    '/nepali/bbc_nepali_radio/liveradio',
+    '/arabic/media-53135426',
+    '/marathi/popular/read',
+    '/gahuza/bbc_gahuza_radio/programmes/p0340x2m',
+    '/nepali/news-50627370',
+    '/arabic/sports-54278377',
+    '/korean/topics/cnwng7v0e54t',
+  ];
+
+  const testResults = await Promise.all(
+    urlsToCheck.map(async url => {
+      const pageSize = exec(
```

Comment:
The results from running the exec curl command are returned in the `stdout` - you need to destruct `stdout` & then rename to `pageSize`
```suggestion
      const { stdout: pageSize } = exec(
```